parfive/downloader.py

```diff
@@ -58,11 +61,10 @@ class Downloader:
     """
 
     def __init__(self, max_conn=5, progress=True, file_progress=True,
-                 loop=None, notebook=None, overwrite=False):
+                 loop=None, notebook=None, overwrite=False, ua_header=None, proxy=None, proxy_auth=None):
```

Comment:
I think I would rather have the proxy info be read from environment variables by default than passed as an argument to the constructor (given it can already be passed as kwargs to `enqueue_file` and I don't want to end up with too many kwargs).